tests/wpt/meta/streams/piping/pipe-through.any.js.ini

```diff
@@ -0,0 +1,142 @@
+[pipe-through.any.shadowrealm-in-shadowrealm.html]
+  expected: ERROR
+
+[pipe-through.any.serviceworker.html]
+  expected: ERROR
+
+[pipe-through.any.worker.html]
+  expected: ERROR
```

Comment:
Note: this PR implements `PipeTo`, not `PipeThrough`(although `PipeThrough` seems like an easy follow-up, since it uses the same underlying mechanism, but it requires `TransformStream`, or at least part of that IDL).